### PR TITLE
Update polish.xml

### PR DESCRIPTION
**Pull request by piotrtrkostrzewski94**
Adaptation for commit 12548b6, 13cd4ac, 35deb8a, aa0be99

### CODE REVIEW

File: PowerEditor/installer/nativeLang/polish.xml [hunk 1]
```diff
@@ -1008,7 +1008,13 @@ Translation note:
 					<Item id="6256" name="Własny kolor"/>
 					<Item id="6258" name="Zastosuj do C0, C1 i EOL z Unicode"/>
 				</Scintillas>
-
+                                <Scintillas2 title="Edycja 2">
+			        <Item id="6521" name="Multi Edycja"/>
+				<Item id="6522" name="Włącz multi edycję (Ctrl+kliknięcie/wybór myszą)"/>
+				<Item id="6523" name="Włącz wybór kolumny do multi edycji"/>	
+			        </Scintillas>
```
[the remark this answers is not in the document]
Comment (piotrtrkostrzewski94) [reply PowerEditor/installer/nativeLang/polish.xml]: I implemented your suggestion. Thank you for info.

**Review by donho — CHANGES_REQUESTED** [review]
Please move the indicated entries from Scintillas tag to Scintillas2 tag.

**Comment by piotrtrkostrzewski94** [comment]
@donho I made the change you requested and I still get the information that Changes requested, could you check my changes?  Have a nice day.

**Review by donho — APPROVED** [review]

**Comment by piotrtrkostrzewski94** [comment]
Thank you very much. Have a nice week.

**Pull request closed without merging**